src/ontology/cl-edit.owl

```diff
@@ -32378,6 +32380,15 @@ SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000101
 SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:RO_0002203 obo:CL_0000127))
 SubClassOf(obo:CL_4042022 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000001307))
 
+# Class: obo:CL_4042023 (substantia nigra dopaminergic neuron)
```

Comment:
ID CLASH

Reply:
will be resolved as the branch is merged.